src/main/kotlin/it/pagopa/wallet/controllers/WalletController.kt

```diff
@@ -51,17 +52,20 @@ class WalletController(
         walletId: UUID,
         exchange: ServerWebExchange?
     ): Mono<ResponseEntity<Void>> {
+        // TODO To be implemented
```

Comment:
what about making those service returning a custom exception to be map into [ExceptionHandler](https://github.com/pagopa/pagopa-wallet-service/blob/531e775fa8e96501a5feaab7e6ba3628935749eb/src/main/kotlin/it/pagopa/wallet/exceptionhandler/ExceptionHandler.kt#L25) to return a 501 http method?

Reply:
That is out of scope for the PR, for now the aim is to lay the groundwork of domain modeling + auditing, the true bussiness logic implementation will be done in the next sprint 👌